src/components/WaveComponent.jsx

```diff
@@ -37,17 +37,13 @@ export class WaveComponent extends React.Component {
         } = this.props;
         if (triggerHandleResize) this._handleResizeTransition();
         if (this.wave) {
-            const isCameraAdjusted = !(
```

Comment:
Let's make sure that when we toggle `isConventionalCellShown` the material gets redrawd

Reply:
Discussed with @tjduigna, conventional cell toggling works as expected.